### PR TITLE
style: Add shared CSS and meganav to datasets docs

### PR DESCRIPTION
**Pull request by stichbury**
## Description
Adding CSS to datasets docs

Preview is here: https://kedro--400.org.readthedocs.build/projects/kedro-datasets/en/400/ and doesn't look quite right 😆 
Please could you take a look @vladimir-mck or @tynandebold ?

## Development notes
Updates to datasets docs files

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the relevant `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Comment by vladimir-mck** [comment]
I see, we need to reduce the font size for the logo/title from 4 to 3rem for it to fit. We can either add a class to it and and reduce the font size for that particular class or we can reduce the font size by 1 for all the docs and keep it consistent. 

@tynandebold  wdyt ?

**Comment by tynandebold** [comment]
Tough one. It's because `kedro-datasets` is a very long string and is giving us a headache.

For it to fit properly and also accompany the words `stable` or even `latest`, the font size will have to be smaller than `3em`:

image

I don't have a good solution for this. I suppose we can have it this size for datasets and I don't think we should change the size of the other titles.

**Review by tynandebold — APPROVED** [review]
Styles are looking good and consistent with the other two projects. Heap data is coming through fine, too.

File: kedro-datasets/docs/source/conf.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 # -- Project information -----------------------------------------------------
 
-project = "kedro-datasets"
+project = "kedro"
```
Comment (astrojuanlu) [inline kedro-datasets/docs/source/conf.py:28]: Wondering if this should still be `kedro-datasets` (sorry I didn't catch that early on)

Comment (stichbury) [reply kedro-datasets/docs/source/conf.py:28]: Oops! Well, next time. At least it's now merged ! And https://docs.kedro.org/projects/kedro-datasets/en/latest/ is looking splendid.

**Pull request merged**